lib/rhc/commands/snapshot.rb

```diff
@@ -30,7 +30,12 @@ def save(app)
 
       raise RHC::DeploymentsNotSupportedException.new if options.deployment && !rest_app.supports?("DEPLOY")
 
-      filename = options.filepath ? options.filepath : "#{rest_app.name}.tar.gz"
+      if options.filepath and File.exists? options.filepath
+        return 0 unless RHC::Helpers.agree "File #{options.filepath} already exists. Do you want to overwrite this file? (yes|no): "
+        filename = options.filepath
+      else
+        filename = options.filepath ? options.filepath : "#{rest_app.name}.tar.gz"
+      end
```

Comment:
I'm aware of that but my reason that since they choose the default name, it's likely that they want to overwrite the old one. However, I think your suggestion cleans up some codes as well so I may as well do it.
Thanks :+1: